index.html

```diff
@@ -5,14 +5,13 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <title>My portfolio mobile version</title>
   </head>
-
   <body>
     <header class="main-header">
       <nav class="nav-bar">
-        <a href="/" class="logo">My Logo</a>
-        <img class="menu" src="/img/Union.svg " alt="" />
+        <a href="index.html" class="logo">My Logo</a>
+        <img class="menu" src="./img/Union.svg " alt="" />
```

Comment:
I think the "alt" attribute should not be left empty. According to HTML  and CSS best practices this is not a good practice.